app/(button)/LoginButton.jsx

```diff
@@ -6,9 +6,11 @@ export default function LoginButton() {
     <Button
       content={
         <span className="flex items-center">
-          Login With <SpotifyIcon className="mx-1" /> Spotify
+          Login With <SpotifyIcon className="mx-1 ml-2" size="45" /> Spotify
         </span>
       }
+      fontSize="text-3xl"
+      padding="py-3 px-6"
       path="/api/login"
     />
```

Comment:
Since we have the path constants now, may as well create one for the /api/login as well?